lib/src/replay/screenshot/screenshot_capturer.dart

```diff
@@ -47,37 +51,33 @@ class ScreenshotCapturer {
     return min(width / srcWidth, height / srcHeight);
   }
 
-  void _updateStatusView(bool shouldSendMetaEvent, RenderObject renderObject,
-      ViewTreeSnapshotStatus statusView) {
-    if (shouldSendMetaEvent) {
-      statusView.sentMetaEvent = true;
-    }
-    _views[renderObject] = statusView;
-  }
-
   Future<Uint8List?> _getImageBytes(ui.Image img) async {
-    final ByteData? byteData =
-        await img.toByteData(format: ui.ImageByteFormat.png);
-    if (byteData == null || byteData.lengthInBytes == 0) {
-      printIfDebug('Error: Failed to convert image to byte data.');
+    try {
+      final ByteData? byteData =
+          await img.toByteData(format: ui.ImageByteFormat.png);
+      if (byteData == null) {
```

Comment:
why removing the `byteData.lengthInBytes == 0` check?

Reply:
sorry, I had copy the function of old version mine